R/test-elements.R

```diff
@@ -1445,7 +1445,10 @@ loop_while <- function(test, logic) {
                 session = session, opt = opt)
     if (!is.scalar.logical(res)) stop("<test> did not return a ",
                                       "logical scalar")
-    if (res) skip_n_pages(state, - (n + 1L))
+    if (res){
+      message(sprintf("loop_while: Skipping %d pages", n))
+      skip_n_pages(state, - (n + 1L))
```

Comment:
Do you want to keep this in, or can it be removed now?

Reply:
Can be removed